spring-pulsar/src/main/java/org/springframework/pulsar/aot/PulsarRuntimeHints.java

```diff
@@ -57,54 +55,191 @@ public void registerHints(RuntimeHints hints, @Nullable ClassLoader classLoader)
+				"org.apache.pulsar.shade.io.netty.channel.DefaultFileRegion",
+				"org.apache.pulsar.shade.io.netty.channel.epoll.NativeDatagramPacketArray$NativeDatagramPacket",
+				"org.apache.pulsar.shade.io.netty.channel.socket.nio.NioDatagramChannel",
+				"org.apache.pulsar.shade.io.netty.channel.socket.nio.NioSocketChannel",
+				"org.apache.pulsar.shade.io.netty.channel.unix.PeerCredentials",
+				"org.apache.pulsar.shade.io.netty.handler.codec.ByteToMessageDecoder",
+				"org.apache.pulsar.shade.io.netty.handler.codec.LengthFieldBasedFrameDecoder",
+				"org.apache.pulsar.shade.io.netty.handler.codec.MessageToMessageDecoder",
+				"org.apache.pulsar.shade.io.netty.handler.codec.MessageToMessageEncoder",
+				"org.apache.pulsar.shade.io.netty.handler.codec.dns.DatagramDnsQueryEncoder",
+				"org.apache.pulsar.shade.io.netty.handler.codec.http.HttpClientCodec",
+				"org.apache.pulsar.shade.io.netty.handler.codec.http.HttpContentDecoder",
+				"org.apache.pulsar.shade.io.netty.handler.codec.http.HttpContentDecompressor",
+				"org.apache.pulsar.shade.io.netty.handler.flush.FlushConsolidationHandler",
+				"org.apache.pulsar.shade.io.netty.handler.stream.ChunkedWriteHandler",
+				"org.apache.pulsar.shade.io.netty.resolver.dns.DnsNameResolver$1",
+				"org.apache.pulsar.shade.io.netty.resolver.dns.DnsNameResolver$3",
+				"org.apache.pulsar.shade.io.netty.resolver.dns.DnsNameResolver$DnsResponseHandler",
+				"org.apache.pulsar.shade.io.netty.util.AbstractReferenceCounted",
+				"org.apache.pulsar.shade.io.netty.util.ReferenceCountUtil",
 				"org.apache.pulsar.shade.io.netty.util.internal.shaded.org.jctools.queues.BaseMpscLinkedArrayQueueColdProducerFields",
+				"org.apache.pulsar.shade.io.netty.util.internal.shaded.org.jctools.queues.BaseMpscLinkedArrayQueueColdProducerFields",
+				"org.apache.pulsar.shade.io.netty.util.internal.shaded.org.jctools.queues.BaseMpscLinkedArrayQueueConsumerFields",
+				"org.apache.pulsar.shade.io.netty.util.internal.shaded.org.jctools.queues.BaseMpscLinkedArrayQueueConsumerFields",
+				"org.apache.pulsar.shade.io.netty.util.internal.shaded.org.jctools.queues.BaseMpscLinkedArrayQueueProducerFields",
+				"org.apache.pulsar.shade.io.netty.util.internal.shaded.org.jctools.queues.BaseMpscLinkedArrayQueueProducerFields",
+				"org.apache.pulsar.shade.io.netty.util.internal.shaded.org.jctools.queues.MpscArrayQueueConsumerIndexField",
+				"org.apache.pulsar.shade.io.netty.util.internal.shaded.org.jctools.queues.MpscArrayQueueConsumerIndexField",
+				"org.apache.pulsar.shade.io.netty.util.internal.shaded.org.jctools.queues.MpscArrayQueueProducerIndexField",
 				"org.apache.pulsar.shade.io.netty.util.internal.shaded.org.jctools.queues.MpscArrayQueueProducerIndexField",
 				"org.apache.pulsar.shade.io.netty.util.internal.shaded.org.jctools.queues.MpscArrayQueueProducerLimitField",
-				"org.apache.pulsar.shade.io.netty.util.internal.shaded.org.jctools.queues.MpscArrayQueueConsumerIndexField")
```

Comment:
I also compressed these into a single block. The value of trying to give the strictest access is not worth the initial effort. That can be left for a later tuning exercise.